tests/unit_tests/lcls_tools/common/measurements/test_emittance_measurement.py

```diff
@@ -0,0 +1,92 @@
+from unittest import TestCase
+from unittest.mock import patch, Mock
+import numpy as np
+
+from lcls_tools.common.devices.reader import create_magnet
+from lcls_tools.common.measurements.emittance_measurement import QuadScanEmittance
+from lcls_tools.common.measurements.screen_profile import ScreenBeamProfileMeasurement
+
+
+class EmittanceMeasurementTest(TestCase):
+    def setUp(self) -> None:
+        self.options = [
+            "TRIM",
+            "PERTURB",
+            "BCON_TO_BDES",
+            "SAVE_BDES",
+            "LOAD_BDES",
+            "UNDO_BDES",
+            "DAC_ZERO",
+            "CALIB",
+            "STDZ",
+            "RESET",
+            "TURN_OFF",
+            "TURN_ON",
+            "DEGAUSS",
+        ]
+        self.ctrl_options_patch = patch("epics.PV.get_ctrlvars", new_callable=Mock)
+        self.mock_ctrl_options = self.ctrl_options_patch.start()
+        self.mock_ctrl_options.return_value = {"enum_strs": tuple(self.options)}
+        self.magnet_collection = create_magnet(area="GUNB")
+        return super().setUp()
+
+    @patch("lcls_tools.common.measurements.emittance_measurement.get_optics")
+    @patch("lcls_tools.common.measurements.emittance_measurement.compute_emit_bmag")
+    @patch(
+        "lcls_tools.common.devices.magnet.Magnet.is_bact_settled",
+        new_callable=Mock,
+    )
+    @patch("epics.PV.put", new_callable=Mock)
+    @patch("lcls_tools.common.devices.magnet.Magnet.trim", new_callable=Mock)
+    def test_quad_scan(self, mock_trim, mock_put, mock_bact_settle, mock_compute_emit_bmag, mock_get_optics):
+        """Test quad scan emittance measurement"""
+        rmat_mock = np.array([
```

Comment:
Would it be possible to store this data in a test file? These values are somewhat distracting.

Reply:
I think I'd do that if I write an integration test, but since the actual numbers don't matter here, I think I can just shorten the numbers so they're not as much of an eyesore.

Reply:
Sounds great!

Reply:
I think it's worth putting some test mats and maybe the twiss data type in our test data folder. @shamin-slac can you write this up as an issue? Thanks in advance.